docs/technical/setup-local-dev-env.md

```diff
@@ -0,0 +1,34 @@
+# Setting up local development environment
+

```

Comment:
Could you please include pre-requisite tools that are required for development.

- Terraform 
- Azure CLI
- Terraform extension (if using VSCode)
- Make

Could also include links to download+install these tools. In the case of the VSCode extension, maybe an extension ID so new developers can search it up.

Reply:
Added section on prerequisite installs needed to set-up terraform locally